app/views/admin/display_ads/index.html.erb

```diff
@@ -34,6 +35,7 @@
             <td><%= link_to display_ad.name, edit_admin_display_ad_path(display_ad) %></td>
             <td><%= display_ad.human_readable_placement_area %></td>
             <td><%= display_ad.display_to %></td>
+            <td><%= display_ad.type_of.titleize %></td>
```

Comment:
**non-blocking comment**: I wonder if it's worth revisiting the fields we display in this table to simplify them. The only thing that comes to mind though would be to merge published and approved as "published & approved" or simply "live" where `published && approved` is true.

Mentioning it because it feels like the fields are being squished a bit, but that might be the case for me (13 inch laptop). Just to reiterate, this is very non-blocking for the sake of this PR 🙂 

![Screen Shot 2022-11-14 at 11 10 20](https://user-images.githubusercontent.com/6045239/201722679-3c5c160d-b261-4b49-a5a5-556e1d78aebb.png)

cc @mirie @benhalpern

Reply:
Took a flyer, it's not a huge help, but maybe a little less squishy?

<img width="995" alt="Screen Shot 2022-11-15 at 09 42 37" src="https://user-images.githubusercontent.com/2077/201879802-c6d2fb48-089d-4dfe-af16-ceb2644e4812.png">

I think the `Edit` and `Destroy` buttons take up a lot of room for how little they are used, but that visual language is fairly consistent across Admin and I'm not sure we want to be arbitrarily changing it.

In one design, we were tucking some of the data attributes in one column, like as bullet-points?

At any rate, I think redesigning this table might be better handled in a separate issue?

Reply:
> At any rate, I think redesigning this table might be better handled in a separate issue?

Definitely makes sense to me! 👍🏼

Reply:
Created a draft issue: https://github.com/forem/forem/issues/18753